src/components/MouseElementTracker.tsx

```diff
@@ -7,6 +7,60 @@ import type { AnyCircuitElement } from "circuit-json"
 import { ifSetsMatchExactly } from "lib/util/if-sets-match-exactly"
 import { pointToSegmentDistance } from "@tscircuit/math-utils"
 
+const detectPrimitives = (
```

Comment:
https://github.com/user-attachments/assets/6b9910fe-5bb3-49cf-98ba-5fe39ab56f8a

```suggestion
const getPrimitivesUnderPoint = (
```